Tests/Tests/STPBankAccountTest.m

```diff
@@ -10,17 +10,76 @@
 
 #import "STPFormEncoder.h"
 #import "STPBankAccount.h"
+#import "STPBankAccount+Private.h"
 
 @interface STPBankAccountTest : XCTestCase
-@property (nonatomic) STPBankAccountParams *bankAccount;
+
+@property (nonatomic) STPBankAccount *bankAccount;
+
 @end
 
 @implementation STPBankAccountTest
 
 - (void)setUp {
+    [super setUp];
     _bankAccount = [[STPBankAccount alloc] init];
 }
 
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+}
+
+#pragma mark - STPBankAccountStatus Tests
+
+- (void)testStatusFromString {
+    XCTAssertEqual([STPBankAccount statusFromString:@"new"], STPBankAccountStatusNew);
+    XCTAssertEqual([STPBankAccount statusFromString:@"NEW"], STPBankAccountStatusNew);
```

Comment:
yeah at first I was hesitant on adding flexibility to the `*fromString` methods but if there is a casing mismatch its probably non-intentional and we should make a best effort return value